API docs for changehc (#276)
cmu-delphi/delphi-epidata: docs/api/covidcast-signals/changehc.md

above.

### Backwards Padding

To help with the reporting delay, we perform the following simple
correction on each location. At each time $t$, we consider the total visit
count. If the value is less than a minimum sample threshold, we go back to the
previous time $t-1$, and add this visit count to the previous total, again
checking to see if the threshold has been met. If not, we continue to move
backwards in time until we meet the threshold, and take the estimate at time
$t$ to be the average over the smallest window that meets the threshold. We
enforce a hard stop to consider only the past 7 days, if we have not yet met the
threshold during that time bin, no estimate will be produced. If, for instance,
at time $t$, the minimum sample threshold is already met, then the estimate
only contains data from time $t$. This is a dynamic-length moving average,
working backwards through time. The threshold is set at 100 observations.

### Smoothing

To help with variability, we also employ a local linear regression filter with a
Gaussian kernel. The bandwidth is fixed to approximately cover a rolling 7 day
window, with the highest weight placed on the right edge of the window (the most

> recent timepoint). Given this smoothing step, the standard error estimate shown
> above is not exactly correct, as the calculation is done post-smoothing.

Reviewer: ```suggestion
recent timepoint).
```

---
title: Change Healthcare
parent: Data Sources and Signals
grand_parent: COVIDcast Epidata API
---

> # Doctor Visits

Reviewer: ```suggestion
# Change Healthcare
```

{: .no_toc}

* **Source name:** `changehc`

> * **Number of data revisions since 19 May 2020:** 0

Reviewer: ```suggestion
* **First issued:** November 4, 2020
* **Number of data revisions since 19 May 2020:** 0
```

* **Date of last change:** Never
* **Available for:** county, hrr, msa, state (see [geography coding docs](../covidcast_geography.md))

This data source is based on information about outpatient visits, provided to us

> by Change Healthcare. Using this outpatient data, we estimate the

Reviewer 2: This should say what Change Healthcare is. A reader who isn't familiar will not know that they process claims on behalf of others, for example, and that's useful context for understanding what will and won't be included in the data

Author: How is the current description?

Reviewer 2: Looks good, but I agree Roni should probably check that the description is accurate and won't cause us problems. You could ping him and show him the draft

Reviewer 2: Looks good, but I agree Roni should probably check that the description is accurate and won't cause us problems. You could ping him and show him the draft

> by health system partners. Using this outpatient data, we estimate the

Reviewer: ```suggestion
by Change Healthcare. Using this outpatient data, we estimate the
```

percentage of COVID-related doctor's visits in a given location, on a given day.

| Signal | Description |
| --- | --- |
| `smoothed_cli` | Estimated percentage of outpatient doctor visits with confirmed COVID-19, based on data from Change Healthcare, smoothed in time using a Gaussian linear smoother |

> | `smoothed_adj_cli` | Same, but with systematic day-of-week effects removed; see [details below](#day-of-week-adjustment) |

Reviewer: These names pending Roni's approval; I've pinged him

Reviewer: Approved

## Table of contents
{: .no_toc .text-delta}

1. TOC
{:toc}

## Lag and Backfill

Note that because doctor's visits may be reported to Change Healthcare
several days after they occur, these signals are typically available with
several days of lag. This means that estimates for a specific day are only
available several days later.

The amount of lag in reporting can vary, and not all visits are reported with
the same lag. After we first report estimates for a specific date, further data

> may arrive about outpatients visits on that date. When this occurs, we issue new

Reviewer: ```suggestion
may arrive about outpatient visits on that date. When this occurs, we issue new
```

outpatient doctor's visits, but not all of them, and so this source only
represents those visits known to them. Their coverage may vary across the United
States, but they report on about 45% of all doctor's visits nationally.

Standard errors are not available for this data source.

Due to changes in medical-seeking behavior on holidays, this data source has
upward spikes in the fraction of doctor's visits that are COVID-related around
major holidays (e.g. Memorial Day, July 4, Labor Day, etc.). These spikes are
not necessarily indicative of a true increase of COVID-like illness in a
location.

## Qualifying Conditions

We receive data on the following two categories of counts:

- Denominator: Daily count of all unique outpatient visits.
- Covid: Daily count of all unique visits with primary ICD-10 code in any of:
{U071, U072, B9721, or B9729}.

## Estimation

### COVID-Like Illness

For a fixed location $i$ and time $t$, let $Y_{it}$
denote the Covid counts and let $N_{it}$ be the
total count of visits (the *Denominator*). Our estimate of the CLI percentage is
given by

$$
\hat p_{it} = 100 \cdot  \frac{Y_{it}}{N_{it}}
$$

> The estimated standard error is:

Reviewer: Is it weird to include this definition if we do not make SE available for this signal in the API?

Author: I wasn't sure. The definition is in the DV docs and SE isn't included there either.

Reviewer 2: Probably was written up while we were deciding if we can or can't include it. May as well remove it from both

Author: Removed
